src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/Tags.scala

```diff
@@ -43,4 +50,34 @@ object Tags {
       }
     }
   }
+
+  case class GetConsignmentId(argName: String) extends ValidateTags {
+    override def validate(ctx: Context[ConsignmentApiContext, _]): BeforeFieldResult[ConsignmentApiContext, Unit] = {
+      argName match {
+        case "addTransferAgreementInput" => ctx.arg(argName)
+      }
+    }
+
+  }
```

Comment:
This isn't being used so you can get rid of it.

Reply:
Fixed